### PR TITLE
intro fixes on product pages

### PR DESCRIPTION
**Pull request by TowerOneka**

### CODE REVIEW

File: site/src/components/pages/ProofMarket/FullCycle/FullCycle.tsx [hunk 1]
```diff
@@ -11,30 +11,51 @@ import ListItem from 'components/ListItem'
...
   className?: string
   data: typeof homePageData.fullCycle
 }
 
+type ArrayElement<A> = A extends readonly (infer T)[] ? T : never
+
 const FullCycle = ({ className, data: { title, description, list, footer } }: FullCycleProps) => {
   const { isMobile } = useViewport()
+
+  const columnList = useMemo(
```
Comment (ukorvl) [inline site/src/components/pages/ProofMarket/FullCycle/FullCycle.tsx:26]: the same.

File: site/src/components/pages/ProofMarket/Win/Win.tsx [hunk 2]
```diff
@@ -11,31 +11,61 @@ import ListItem from 'components/ListItem'
...
 
 type WinProps = {
   className?: string
   data: typeof homePageData.win
 }
 
+type ArrayElement<A> = A extends readonly (infer T)[] ? T : never
+
 const Win = ({ className, data: { title, description, content, footer } }: WinProps) => {
+  const columnList = useMemo(
```
Comment (ukorvl) [inline site/src/components/pages/ProofMarket/Win/Win.tsx:24]: the same

**Review by ukorvl — APPROVED** [review]
Overall LGTM

File: site/src/components/pages/ProofMarket/FullCycle/FullCycle.tsx (hunk 1 above)
Comment (TowerOneka) [reply site/src/components/pages/ProofMarket/FullCycle/FullCycle.tsx:26]: Merged from page branch

I will update and merge with master now

**Pull request merged**